pygerduty/v2.py

```diff
@@ -184,6 +184,10 @@ def list(self, **kwargs):
                     break
 
                 for item in this_paginated_result:
+                    # add items to seen_items only if id is present in response
+                    # oncalls does not have a id in response
+                    if not hasattr(item, 'id'):
+                        continue
```

Comment:
because of this, querying for a single item is failing. It returns an empty result.  

{code}
p.oncalls.show(schedule_ids=['fooo']) 
{code}
is returning an empty result.